reframe/frontend/cli.py

```diff
@@ -53,6 +53,13 @@ def format_check(check, detailed):
     return '\n'.join(lines)
 
 
+def format_env(envvars):
+    ret = 'ReFrame environment variables:\n'
+    notset = "<not set>"
+    ret += '\n'.join([f'    {e}={os.environ.get(e, notset)}' for e in envvars])
```

Comment:
```suggestion
    ret += '\n'.join([f'  {e}={os.getenv(e, notset)}' for e in envvars])
```

To match the  following section.

Reply:
Print also the `RFM_INSTALL_PREFIX`. This is not in the environment variables returned by the argument parser.

Reply:
@vkarak should I keep the order or just print it above every env variable?

Reply:
Since you sort them, it'd be betted if that one was in the right order. I guess it's more correct in any case to sort them here (the strings) instead of sorting them in `env_vars`. Also you don't need the list here. You can call `join()` on a generator.